content/docs/authorization/deployment/helm/_index.md

```diff
@@ -133,13 +133,9 @@ Karavictl commands and intended use can be found [here](../../cli/).
 
 ## Configuring the CSM Authorization Proxy Server
 
-The storage administrator must first configure the proxy server with the following:
-- Storage systems
-- Tenants
-- Roles
-- Role bindings
+The first part of CSM for Authorization deployment is to configure the proxy server. This is controlled by the Kubernetes storage administrator.
```

Comment:
Storage administrator, not specific to kubernetes.

Reply:
fixed.